pages/common/buzzphrase.md

```diff
@@ -0,0 +1,16 @@
+# buzzphrase
+
+> Node command line tool to output a random buzzphrase.
+> More information: <https://github.com/atomantic/buzzphrase>.
+
+- Generate a string of three random adjective, past-tense verb and plural noun:
```

Comment:
Perhaps this will read better?
```suggestion
- Generate a string of three random adjectives, past-tense verbs and plural nouns:
```

Reply:
Thanks. I reread the sentence and thought it would be better to write like this:

`Generate a string of three random phrases containing an adjective, a past tense verb and a plural noun:`

Reply:
Sounds good! Let's do that then.

Reply:
Ok. I'll know :)